app/jsontableschema/constraints.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
```

Comment:
Files you are interested in:
app/storage/*
app/models/*
app/services/cases.js
app/services/trials.js